onward/app/models/dotcomponents/DotcomponentsOnwardsModels.scala

```diff
@@ -34,18 +34,18 @@ object RichLink {
   implicit val writes = Json.writes[RichLink]
 }
 
-object OnwardsUtils {
+// object OnwardsUtils {
```

Comment:
do we tend to keep commented out code for posterity's sake?

Reply:
No- will delete!